installer/Migration/V5_6_0/Migration.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA
+ */
+
+namespace OrangeHRM\Installer\Migration\V5_6_0;
+
+use Doctrine\DBAL\Schema\ForeignKeyConstraint;
+use OrangeHRM\Installer\Util\V1\AbstractMigration;
+
+class Migration extends AbstractMigration
+{
+    /**
+     * @inheritDoc
+     */
+    public function up(): void
+    {
+        $this->getSchemaHelper()->dropForeignKeys('ohrm_i18n_translate', ['langStringId']);
```

Comment:
Add this scenarios to test scope; language customization was introduced in [4.6](https://github.com/orangehrm/orangehrm/wiki/Release-Note#orangehrm-version-46)

- Test upgraded instance from below 4.6 version
- Test upgraded instance from 4.6 
- Test upgraded instance from 4.6 or above version which has language customization

Mainly want to focus on `langStringId` foreign key drop